### PR TITLE
chore: update Playwright to 1.34

### PR DESCRIPTION
**Pull request by worldomonation**
## Proposed Changes

This PR updates Playwright to 1.34 to keep up with the releases.

## Testing Instructions

Ensure the following build configurations are passing:
  - [ ] Gutenberg E2E Simple production (desktop)
  - [ ] Gutenberg E2E Simple production (mobile)
  - [x] Calypso E2E (desktop)
  - [ ] Calypso E2E (mobile)
  - [x] Pre-Release E2E

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: test/e2e/specs/plugins/plugins__install-remove.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ describe( DataHelper.createSuiteTitle( 'Jetpack: Plugin' ), function () {
 
 		page = await browser.newPage();
 
-		const testAccount = new TestAccount( 'jetpackRemoteSiteUser' );
+		const testAccount = new TestAccount( 'atomicUser' );
```
Comment (karenroldan) [inline test/e2e/specs/plugins/plugins__install-remove.ts]: **question:** Is this change intended to be part of this PR? I see that data of `jetpackRemoteSiteUser` are still used within the test.

Comment (worldomonation) [reply test/e2e/specs/plugins/plugins__install-remove.ts]: No. Not sure how that made it in this change when I selectively picked files to include.

Comment (worldomonation) [reply test/e2e/specs/plugins/plugins__install-remove.ts]: Pushed revision that removes the change. Thanks for noticing!

**Review by karenroldan — APPROVED** [review]
LGTM 🚀  all tests passed. 

note: I had to rerun tests for Gutenberg simple E2E (desktop). Seems the validation on the Instagram block had a hiccup.
